Sources/Common/Core/LookupTable/index.js

```diff
@@ -175,6 +177,66 @@ function vtkLookupTable(publicAPI, model) {
     } // alpha blending
   };
 
+  publicAPI.setTable = (newTable) => {
+    if (Array.isArray(newTable) && newTable !== model.table) {
+      if (newTable.length % 4 !== 0) {
+        vtkErrorMacro('setTable: table must have 4-component entries');
+        return false;
+      }
+      model.table = newTable;
```

Comment:
Do we care if this is the same reference, or should I copy newTable?

Reply:
copy